main.py

```diff
@@ -91,26 +92,37 @@ def delete_label_file(self):
         label_filename = os.path.splitext(self.image_filename)[0] + ".txt"
         if os.path.exists(label_filename):
             os.remove(label_filename)
-    def load_labels(self, image_filename, existing_labels):
+
+
+    def load_labels(self, image_filename, existing_labels, rollover_label):
```

Comment:
Isn't rollover_label a member variable---therefore directly accessed via self.rolling_over_labels.

On a side note (unrelated to your code), is existing_labels used in this function? If not, we could remove it.